corfudb-tools/src/main/java/org/corfudb/compactor/CorfuStoreCompactorMain.java

```diff
@@ -0,0 +1,416 @@
+/* *****************************************************************************
+ * Copyright (c) 2016-2019. VMware, Inc.  All rights reserved. VMware Confidential
+ * ****************************************************************************/
+package org.corfudb.compactor;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Optional;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.runtime.CorfuCompactorManagement.StringKey;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.DistributedCompactor;
+import org.corfudb.runtime.collections.*;
+import org.corfudb.runtime.proto.RpcCommon.TokenMsg;
+import org.corfudb.util.GitRepositoryState;
+import org.docopt.Docopt;
+
+import com.google.protobuf.Message;
+
+import org.corfudb.runtime.CorfuStoreMetadata.ProtobufFileDescriptor;
+import org.corfudb.runtime.CorfuStoreMetadata.ProtobufFileName;
+import org.corfudb.runtime.CorfuStoreMetadata.TableDescriptors;
+import org.corfudb.runtime.CorfuStoreMetadata.TableMetadata;
+import org.corfudb.runtime.CorfuStoreMetadata.TableName;
+import org.corfudb.runtime.exceptions.SerializerException;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.ProtobufSerializer;
+
+import java.util.Date;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.util.Set;
+import java.util.UUID;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import static org.corfudb.runtime.view.TableRegistry.getTypeUrl;
+
+@Slf4j
+public class CorfuStoreCompactorMain {
+
+    private static CorfuRuntime corfuRuntime;
+    private static CorfuStore corfuStore;
+    private static CorfuStoreCompactor corfuCompactor;
+
+    private static final String CORFU_SYSTEM_NAMESPACE = "CorfuSystem";
+
+
+    private static UUID thisNodeUuid;
+
+    private static ISerializer protobufSerializer;
+
+    private static Table<StringKey, TokenMsg, Message> checkpoint;
+    private static Table<StringKey, TokenMsg, Message> previousToken;
+
+    // Reduce checkpoint batch size due to disk-based nature and smaller compactor JVM size
+    private static final int NON_CONFIG_DEFAULT_CP_MAX_WRITE_SIZE = 1 << 20;
+
+    private static final int DEFAULT_CP_MAX_WRITE_SIZE = 25 << 20;
+
+    private static List<String> hostname = new ArrayList<>();
+    private static int port;
+    private static String runtimeKeyStore;
```

Comment:
All these parameters seems to be taken from CorfuRuntimeHelper^ is it possible to reuse the helper or CorfuRuntimeParameters class?